src/input-range-select/input-range-select.tsx

```diff
@@ -134,6 +136,12 @@ export const InputRangeSelect = <T, V>({
         }
     };
 
+    const handleClear = (event: { stopPropagation: () => void }) => {
```

Comment:
use the inbuilt React event type

```suggestion
    const handleClear = (event: React.MouseEvent) => {
```